voxblox/include/voxblox/core/common.h

```diff
@@ -78,6 +78,7 @@ typedef AlignedVector<Triangle> TriangleVector;
 typedef kindr::minimal::QuatTransformationTemplate<FloatingPoint>
     Transformation;
 typedef kindr::minimal::RotationQuaternionTemplate<FloatingPoint> Rotation;
+typedef Eigen::Quaternion<FloatingPoint> Quaternion;
```

Comment:
Would it be better to use kindr::minimal::RotationQuaternionTemplate<FloatingPoint>::Implementation
that way all rotations stay as minkindr types